controls/datepicker/dateonly.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Date Only Support
+page_title: .NET MAUI Date Picker Documentation - Date Only Support
+description: Set date only type in the Telerik UI for .NET MAUI DatePicker.
+position: 4
+slug: datepicker-dateonly
+---
+
+# Date Only Support in .NET MAUI DatePicker
+
+The DatePicker has a support for `DateOnly`. You can set a `DateOnly` value to the `MinimumDate`, `MaximumDate`, `Date` and `DefaultHighlightedDate` properties by using the Telerik `DateOnlyToDateTimeConverter `converter. The converter converts `System.DateOnly` to `System.DateTime` and converts back `System.DateTime` to `System.DateOnly`.
```

Comment:
- Describing a simple potential use case allows you to set the context for the article that describes a specific feature.
- This comment also applies to the rest of the articles in this PR.

```suggestion
If your application needs to store or manipulate only the dates without times, you can take advantage of the `DateOnly` struct when working with the DatePicker control.

You can set a `DateOnly` value to the `MinimumDate`, `MaximumDate`, `Date`, and `DefaultHighlightedDate` properties by using the Telerik `DateOnlyToDateTimeConverter `converter. The converter converts `System.DateOnly` to `System.DateTime` and `System.DateTime` to `System.DateOnly`.
```